documentation/commands/ship.md

```diff
@@ -26,6 +26,9 @@ into the main branch, resulting in linear history on the main branch.
 Only shipping of direct children of the main branch is allowed.
 To ship a nested child branch, all ancestor branches have to be shipped or killed.
 
+If you have a pull request for your branch on GitHub and the environment variable `GIT_TOWN_GITHUB_TOKEN` set to a GitHub personal access token with access to your repositories,
```

Comment:
Initially giving the users the git config command should be enough to get this feature released. Later I think a command to set this up properly would be nice.

- `git config ship-strategy`: asks for the Github token, verifies it, and if it works, writes it to the Git config